build-system/routes/analytics.js

```diff
@@ -0,0 +1,46 @@
+/**
+ * Copyright 2019 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+const router = require('express').Router();
+
+router.use('/rewriter', (req, res) => {
+  const body = JSON.parse(req.body);
+  if (body.vars && body.vars.url) {
+    const requestsConfig = {
+      requests: {
+        endpoint: body.vars.url,
+      },
+    };
+    Object.assign(body, requestsConfig);
+  }
+
+  const extraUrlParams = {
+    extraUrlParams: {
+      testId: 12358,
+      rewritten: true,
+    },
+  };
+  Object.assign(body, extraUrlParams);
+  res.json(body);
+});
+
+router.use('/:type', (req, res) => {
+  console.log('Analytics event received: ' + req.params.type);
```

Comment:
consider to use `log` method in amp4test.js , so it can be turned on/off with the --verbose flag

Reply:
Done.